### ISSUE TITLE
2022-03-04 Release(s) Plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#1367]
## Minor Release

2.0.0 or 2.0.0-alpha1?

### CSS, markup or template changes - Changes to visual appearance of a website:
- [x] #1335
- [x] #1353

### APIs - New internal APIs or API improvements with backwards compatibility
- [x] #1277
- [x] #1279
- [x] #1319
- [x] #1321
- [x] #1362
- [x] #1372

### Drupal contrib projects - Minor release updates
- [x] #1258

## Patch Release(s)

2.1.3, 2.0.19(?)

### Bug fixes

- [x] #1314
- [x] #1226
- [x] #1342
- [x] #1360
- [x] #1347
- [x] #1334

### Critical institutional link changes

- [x] #1326

### Drupal contrib projects - Addition of new modules

- [x] #1320
- [x] #1315
- [x] #1337

### APIs - New internal APIs or API improvements with backwards compatibility
(Special case necessitates inclusion in patch release(s))
- [x] #1351 (**2.1.x only**)
- [x] #1352 (**2.0.x only**)
- [x] #1332 (**2.1.x only**)

## Uncategorized changes

- [x] #1325
- [x] #1355

## Categories that potentially need to be added to RELEASES.md
- Documentation updates
- Development tool updates
- Continuous integration tool updates

**Comment by joeparsons** [comment]
# 2.0.19

## Pick list
```
git cherry-pick ec4bb07e38209d6eaefc236a437fc209968630bc
git cherry-pick c7c5d918669ab2de673dbf5a42d3252e512f07b0
git cherry-pick df71df5aef1bedfe9048b6ab5c84cedcce92a6ef
git cherry-pick caec8c5750e126107b4f14d981af0302ff57e722
git cherry-pick ab1ff6ffaec8ce67612cea99e9637b93ace3c7f1
git cherry-pick db0378c235fd504ed20b8d0f7248cf60bd2edf77
git cherry-pick 5baa57905649a937430d119ff5295ae8d27c0f94
git cherry-pick c4751cb93a2b3d8def3049709bdda56ba11051f1
git cherry-pick b33eba6ae2b0fccca9642f8ff04c2a81dc5181f9
git cherry-pick 92be93ef05ed757648361bce84c3012b821449ac
git cherry-pick 2bf997bfcd9f195e93851ce0a47bc268d7d9358f
git cherry-pick 64f86f621354dc75cbbccf0e5b3cbdb29c11c9e5
git cherry-pick 2cb78be73346845bb22e679ed025b025cc1ddebe

```

## Release notes snippet
`git log 2.0.18..HEAD --pretty='format:- %s'`

```
- Closes #1333: Configure system mail interface to use SMTP during install. (#1334)
- Closes #1331: Make sidebar not overlap text for Text with Background (#1360)
- Fixes #1354 Lando database doesn't play nicely with M1 Mac. (#1355)
- Closes #1336: Add antibot module to az_security (#1337)
- Closes #1344 Migration: Extra (blank) Text Boxes are showing up (#1347)
- Closes #1306: Update PR template for better alignment with our minor/patch release definitions
- Fixes #1328: replace non-breaking space between first and last name (#1342)
- Closes #960: Adding exclude_node_title per node setting migration. (#1319)
- Closes #1309 Adding quick node clone (#1315)
- Closes #1225: Removing setting the class array as a string everywhere, and fixing bottom spacing issue. (#1226)
- Closes #1153: Change red bar wordmark link to https (#1326)
- Closes #1296: Use views label instead of rendered views inside paragraph previews.
- Closes #682: Added Better Exposed Filters module. (#1320)
- #1350: 2.0.x backport of flexible update hook numbering scheme implementation. (#1352)
```

**Comment by joeparsons** [comment]
# 2.1.3

## Pick list
```
git cherry-pick ec4bb07e38209d6eaefc236a437fc209968630bc
git cherry-pick c7c5d918669ab2de673dbf5a42d3252e512f07b0
git cherry-pick df71df5aef1bedfe9048b6ab5c84cedcce92a6ef
git cherry-pick caec8c5750e126107b4f14d981af0302ff57e722
git cherry-pick ab1ff6ffaec8ce67612cea99e9637b93ace3c7f1
git cherry-pick db0378c235fd504ed20b8d0f7248cf60bd2edf77
git cherry-pick 5baa57905649a937430d119ff5295ae8d27c0f94
git cherry-pick c4751cb93a2b3d8def3049709bdda56ba11051f1
git cherry-pick b33eba6ae2b0fccca9642f8ff04c2a81dc5181f9
git cherry-pick 92be93ef05ed757648361bce84c3012b821449ac
git cherry-pick 2bf997bfcd9f195e93851ce0a47bc268d7d9358f
git cherry-pick 64f86f621354dc75cbbccf0e5b3cbdb29c11c9e5
git cherry-pick 54ee481a31a35cf2722b2fc1c9e3ad23239e4a05
git cherry-pick 2cb78be73346845bb22e679ed025b025cc1ddebe
git cherry-pick 55e77f62f1581e7c413203ec0649d87a36b92e0e
```

## Release notes snippet
`git log 2.1.2..HEAD --pretty='format:- %s'`

```
- Closes #1317: Add update hook to delete problematic legacy global footer menu blocks. (#1332)
- Closes #1333: Configure system mail interface to use SMTP during install. (#1334)
- Closes #1350: Implement more flexible update hook numbering scheme. (#1351)
- Closes #1331: Make sidebar not overlap text for Text with Background (#1360)
- Fixes #1354 Lando database doesn't play nicely with M1 Mac. (#1355)
- Closes #1336: Add antibot module to az_security (#1337)
- Closes #1344 Migration: Extra (blank) Text Boxes are showing up (#1347)
- Closes #1306: Update PR template for better alignment with our minor/patch release definitions
- Fixes #1328: replace non-breaking space between first and last name (#1342)
- Closes #960: Adding exclude_node_title per node setting migration. (#1319)
- Closes #1309 Adding quick node clone (#1315)
- Closes #1225: Removing setting the class array as a string everywhere, and fixing bottom spacing issue. (#1226)
- Closes #1153: Change red bar wordmark link to https (#1326)
- Closes #1296: Use views label instead of rendered views inside paragraph previews.
- Closes #682: Added Better Exposed Filters module. (#1320)
```

**Comment by joeparsons** [comment]
# 2.2.0

## Steps required
(once everything to be included in `2.2.0` has been merged)

### az_quickstart repository
1.  Create & merge PR that updates the composer branch alias for `dev-main` to be `2.3.x-dev`. This should be back-ported to all active release branches after it is merged (`2.0.x`, `2.1.x`).
3. Create patch releases for previous active release branches (`2.0.19`, `2.1.3`)
4. Create `2.2.x` branch from `main`
5. Create & merge PR that updates branch references in CI and local development config files to use `2.2.x` in the `2.2.x` branch
6. Create `2.2.0` release

### az-quickstart-scaffolding repository
1. Create `2.2.x` branch from `main`
2. Create & merge PR that updates `az_quickstart` version used in `main` branch to be the new branch alias (`~2.3`)
3. Create & merge PR that updates branch references in CI and local development config files to use `2.2.x` in the `2.2.x` branch

**Comment by joeparsons** [comment]
Releases created:
https://github.com/az-digital/az_quickstart/releases/tag/2.2.0
https://github.com/az-digital/az_quickstart/releases/tag/2.1.3
https://github.com/az-digital/az_quickstart/releases/tag/2.0.19
